pages/common/meshlabserver.md

```diff
@@ -0,0 +1,23 @@
+# meshlabserver
+
+> Command line interface for the MeshLab 3D mesh processing software.
+
+- Convert an STL file to an OBJ file:
+
+`meshlabserver -i {{input.stl}} -o {{output.obj}}`
+
+- Convert a WRL file to a OFF file, inlcuding the vertex and face normals in the output:
```

Comment:
Ok, after some investigation I figured out that the `-o -m` syntax was changed form `-om` in [r6378](https://sourceforge.net/p/meshlab/code/6378/) ([11b92d3](https://github.com/svn2github/meshlab/commit/11b92d3) in the git mirror), committed on 2014-04-18 -- only days after the latest release, [v1.3.3](https://sourceforge.net/projects/meshlab/files/meshlab/)! So for now we must stick with `-om`.

The abbreviation seems to mean "output mesh (options)". Should I add "mesh" to this command description?